.env.example

```diff
@@ -3,3 +3,4 @@ VITE_PORTALS_BASE_URL=https://api.portals.fi
 VITE_PORTALS_API_KEY=copy me from web
 VITE_BEBOP_API_KEY=request me
 VITE_EVM_MNEMONIC=your seed here - for server testing only
+VITE_PROJECT_ID=request me, or create a reown cloud account (eventually will need to use ShapeShift's)
```

Comment:
i.e created own reown cloud account for the time being, took like a min so meh

Reply:
I'll get this created and add in a follow-up PR 🙏